mysite/lti/views.py

```diff
@@ -0,0 +1,119 @@
+import pickle
+import oauth2
+import json
+import logging
+from django.template import RequestContext
+from django.core.urlresolvers import reverse
+from django.views.decorators.csrf import csrf_exempt
+from ims_lti_py.tool_provider import DjangoToolProvider
+from django.shortcuts import render_to_response, redirect
+
+from lti.models import LTIUser
+from lti import app_settings as settings
+
+
+ROLES_MAP = {
+    'Instructor': 'prof',
+    'Leaner': 'student',
```

Comment:
Yes, it's a typo. I can fix this but I will need a write access to branch `ltipsa`. 
P.S. I also need a write access for fixing your future comments.